action.yml

```diff
@@ -32,6 +32,13 @@ inputs:
     description: Mark the created GitHub release as 'latest'
     required: false
     default: true
+outputs:
+  changed_charts:
+    description: "A comma-separated list of charts that were released on this run. Will be an empty string if no updates were detected, will be unset if `--skip_packaging` is used: in the latter case your custom packaging step is responsible for setting its own outputs if you need them."
+    value: ${{ steps.release.outputs.changed_charts }}
+  chart_version:
+    description: "The version of the most recently generated charts; will be set even if no charts have been updated since the last run."
+    value: ${{ steps.release.outputs.chart_version }}
 
 runs:
   using: composite
```

Comment:
in the steps need to add the id, which is missing, should be something like

```
  steps:
    - id: release
      run: |
        owner=$(cut -d '/' -f 1 <<< "$GITHUB_REPOSITORY")
        repo=$(cut -d '/' -f 2 <<< "$GITHUB_REPOSITORY")
....
```

Reply:
Done!